apps/public-docsite-v9/package.json

```diff
@@ -12,7 +12,7 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook",
+    "start": "yarn storybook:docs",
```

Comment:
If I want to run the previous version of the `start` script how do I do that now?

Reply:
running yarn storybook from the package root would do it

Reply:
there just isn't much reason to do so. the chromatic command will publish the --docs version